### DISCUSSION TITLE
Add Collapse to Visibility enumeration.

### DISCUSSION THREAD

**Discussion by Lasoty** [Megabit/Blazorise#3418] [Ideas]
I suggest adding the "Collapse" item to the Visibility enumeration, which will work similarly to WPF, that is: if Visibility == Visibility.Collapse then the element does not render on the page and does not take up the allocated space.

**Comment by stsrki** [comment]
Visibility enum is used to control the element visibility while still maintaining its space. To hide the element and remove its occupied space you can already use a Display parameter, eg. `Display="Display.None"`.

**Reply by Lasoty** [reply]
Unfortunately static types cannot be used as parameters. I need some idea to dynamically hide frontend element.

**Reply by Lasoty** [reply]
Sorry, my fault. I should to use `IFluentDisplayWithDisplayOnBreakpointWithDirection` type to store display value.
